### PR TITLE
Doc changes and improvements

### PR DESCRIPTION
**Pull request by pecameron**
Major changes:
ovnkube - sort options to same order as --help
ovn_k8s - document the config file.

### CODE REVIEW

**Comment by pecameron** [comment]
@dcbw @rajatchopra I updated the man pages based on PR 228 and added a new man page for ovn_k8s.conf. Could we get this in before making the next version? PTAL

File: docs/ovn_k8s.5 [hunk 1]
```diff
@@ -0,0 +1,94 @@
+.TH OVN_K8S.CONF "5" "Jan 2018" "ovn-kubernetes" "OVN-KUBERNETES Configuration File"
```
Comment (dcbw) [inline docs/ovn_k8s.5]: Should this manpage be called ovn_k8s.conf.5?

**Comment by pecameron** [comment]
@dcbw @rajatchopra I change the file name ovn_k8s.5 to ovn_k8s.conf.5 and fixed the spec file. PTAL

**Comment by pecameron** [comment]
@dcbw @rajatchopra now that PR 228 is merged, can we get this finished up? I need this in to finish up PR 230 which is part of the basis of the ovn daemonset work. Thanks!

**Comment by rajatchopra** [comment]
@pecameron We should wait for #245, though I am not expecting many changes

**Pull request merged**